dart/lib/src/sentry_options.dart

```diff
@@ -218,7 +218,14 @@ class SentryOptions {
   /// variables. This is useful in tests.
   EnvironmentVariables environmentVariables = EnvironmentVariables.instance();
 
-  /// When enabled, all the threads are automatically attached to all logged events (Android).
+  /// When enabled, the current isolate will be attached to the event.
+  /// This only applies to Dart:io platforms and only the current isolate.
+  /// Other isolates can't be read.
+  ///
+  /// When running on web, this option has no effect at all.
+  ///
+  /// When running in the Flutter context, this enables attaching of threads
+  /// for native events, if supported for the native platform.
```

Comment:
I'd add the note that it's only Android at the moment tho.

Reply:
Are there any docs I could link to? Then no one has to remember to update this doc comment if that changes.